.changeset/wise-llamas-drop.md

```diff
@@ -0,0 +1,137 @@
+---
+'@graphql-yoga/plugin-jwt': major
+---
+
+Re-write for the JWT plugin. This plugin can be configured now with multiple providers, lookup locations, token verification, and more. 
+
+The version has better version coverage, and it provides an improved API for configuring provider and custom behaviors.
+
+## Breaking Change: New Plugin Configuration
+
+### Signing key providers
+
+❌ The `signingKey` option has be removed.
+❌ The `jwksUri` + `jwksOpts` options has been removed.
+✅ Multiple signing key providers and support for fallbacks (`singingKeyProviders[]`).
```

Comment:
```suggestion
✅ Multiple signing key providers and support for fallbacks (`signingKeyProviders[]`).
```
I think this type is copy-pasted in a lot of places.

Reply:
fixed, thanks